package.json

```diff
@@ -38,13 +39,16 @@
     "gulp-uglify": "^0.3.1",
     "gulp-useref": "^0.5.0",
     "gulp-util": "^3.0.0",
-    "gulp-protractor": "^0.0.11",
     "ink-docstrap": "^0.4.12",
+    "jasmine-core": "^2.1.3",
```

Comment:
I understand, no problem I really just wrote one test to get the whole thing and up running, no time wasted.